.github/workflows/ci3.yml

```
@@ -13,9 +13,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v1
-      - uses: actions/checkout@v1 # Comment that Renovate bot will remove later on
-      - uses: actions/checkout@1e204e9a9253d643386038d443f96446fa156a97 # renovate: tag=v2.3.5 # Comment that Renovate bot has moved Comment that Renovate bot will remove later on
+      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3# Comment that Renovate bot will remove later on
```

```suggestion
      - uses: actions/checkout@v3 # Comment that Renovate bot will remove later on
```
The space should be retained